### PR TITLE
Add Boba Avalanche docs

### PR DESCRIPTION
**Pull request by boyuan-chen**
:clipboard: Add associated issues, tickets, docs URL here.

## Overview

Describe what your Pull Request is about in a few sentences.

Add Boba Avalanche docs

## Changes

Describe your changes and implementation choices. More details make PRs easier to review.

- Add docs

## Testing

Describe how to test your new feature/bug fix and if possible, a step by step guide on how to demo this.

### CODE REVIEW

**Review by InoMurko — APPROVED** [review]

File: boba_documentation/developer/network-avalanche.md [hunk 1]
```diff
@@ -57,26 +59,48 @@ For **secondary addresses**, such as L2 Tokens and Messengers, please see the [B
...
-| ------------- | -------------------------- |
-| ChainID       | 43288                      |
-| RPC Read      | [https://tbd](https://tbd) |
-| Write RPC     | [https://tbd](https://tbd) |
-| Gateway       | [https://tbd](https://tbd) |
-| Blockexplorer | [https://tbd](https://tbd) |
-| Websocket     | [wss://tbd](wss://tbd)     |
+|               |                                                              |
+| ------------- | ------------------------------------------------------------ |
+| ChainID       | 43288                                                        |
```
Comment (InoMurko) [inline boba_documentation/developer/network-avalanche.md:101]: 43114

**Review by InoMurko — CHANGES_REQUESTED** [review]

File: boba_documentation/developer/network-avalanche.md [hunk 2]
```diff
@@ -57,26 +59,48 @@ For **secondary addresses**, such as L2 Tokens and Messengers, please see the [B
...
+
 ## Boba Avalanche Addresses (tbd)
 
-For **primary contracts and addresses** see [packages/contracts/deployments/bobaavalanche/README.md](../../packages/contracts/deployments/bobaavalanche/README.md)
+For **primary contracts and addresses** see [packages/contracts/deployments/bobaavax/README.md](../../packages/contracts/deployments/bobaavax/README.md)
 
-For **secondary addresses**, such as L2 Tokens and Messengers, please see the [Boba Avalanche address registration dump](../../packages/boba/register/addresses/addressesBobaOpera_0xTBATBATBATBA.json).
+For **secondary addresses**, such as L2 Tokens and Messengers, please see the [Boba Avalanche address registration dump](../../packages/boba/register/addresses/addressBobaAvax_0x00220f8ce1c4be8436574e575fE38558d85e2E6b.json).
 
 ## Boba Avalanche Links and Endpoints (tbd)
```
Comment (InoMurko) [inline boba_documentation/developer/network-avalanche.md]:
```suggestion
## Boba Avalanche Links and Endpoints 
```

File: boba_documentation/developer/network-avalanche.md [hunk 3]
```diff
@@ -57,26 +59,48 @@ For **secondary addresses**, such as L2 Tokens and Messengers, please see the [B
...
+
+ The bobabeam's docker-compose file is in [`boba-community/boba-node/docker-compose-bobaavax.yml`](https://github.com/bobanetwork/boba/tree/alt-l1/boba_community/boba-node)
+
+ ```bash
+ $ docker compose -f docker-compose-bobaavax.yml pull
+ $ docker compose -f docker-compose-bobaavax.yml up
+ ```
+
+
 ## Boba Avalanche Addresses (tbd)
```
Comment (InoMurko) [inline boba_documentation/developer/network-avalanche.md]:
```suggestion
## Boba Avalanche Addresses 
```

File: boba_documentation/developer/network-avalanche.md [hunk 4]
```diff
@@ -57,26 +59,48 @@ For **secondary addresses**, such as L2 Tokens and Messengers, please see the [B
 | ------------- | ------------------------------------------------------------ |
 | ChainID       | 43288                                                        |
```
Comment (InoMurko) [inline boba_documentation/developer/network-avalanche.md]: this is wrong:
```
curl https://testnet.avax.boba.network   -X POST   -H "Content-Type: application/json"   --data '{"method":"eth_chainId","params":[],"id":1,"jsonrpc":"2.0"}'
{"jsonrpc":"2.0","result":"0x10e8","id":1}
```

File: boba_documentation/developer/network-avalanche.md (hunk 1 above)
Comment (boyuan-chen) [reply boba_documentation/developer/network-avalanche.md:101]: The chain Id for Bobaavax testnet is 4328.

**Pull request merged**